fvm/transactionEnv.go

```diff
@@ -950,3 +951,35 @@ func (e *TransactionEnv) Commit() ([]programs.ContractUpdateKey, error) {
 	}
 	return e.contracts.Commit()
 }
+
+func (e *TransactionEnv) BLSVerifyPOP(pk *runtime.PublicKey, sig []byte) (bool, error) {
```

Comment:
These methods seem to be redundant with `ScriptEnv` methods, it would be good to factorize this code to make it use the same code (better maintainability).